test/integration/bad/embedded_laplace/stanc.expected

```diff
@@ -548,48 +548,51 @@ Semantic error in 'missing_args7.stan', line 18, column 24 to line 19, column 23
-for the likelihood:
-Expected 4 arguments but got 3 arguments.
-Expected the arguments to start with:
-(array[] int, array[] int, vector, vector)
+Ill-typed arguments supplied to function 'laplace_latent_neg_binomial_2_log_rng'.
+The valid signature of this function is
+laplace_latent_neg_binomial_2_log_rng(array[] int, array[] int, vector,
+  vector, vector, (T_k...) => matrix, tuple(T_k...))
+However, we recieved the types:
+(data array[] int, data array[] int, data vector, data vector)
+Typechecking failed after checking the first 4 arguments. Please ensure you
+are passing enough arguments and that the 5th is a function.
 [exit 1]
   $ ../../../../../install/default/bin/stanc missing_args8.stan
-Semantic error in 'missing_args8.stan', line 2, column 12 to column 59:
+Semantic error in 'missing_args8.stan', line 2, column 12 to column 65:
    -------------------------------------------------
      1:  model {
-     2:    target += laplace_marginal_tol_poisson_log_lpmf({1}, {1});
+     2:    target += laplace_marginal_tol_poisson_log_lpmf({1}, {1}, [0]');
                      ^
      3:  }
    -------------------------------------------------
 
-Ill-typed arguments supplied to function 'laplace_marginal_tol_poisson_log_lpmf'
-for the likelihood:
-Expected 3 arguments but got 2 arguments.
-Expected the arguments to start with:
-(array[] int, array[] int, vector)
+Ill-typed arguments supplied to function 'laplace_marginal_tol_poisson_log_lpmf'.
+The valid signature of this function is
+laplace_marginal_tol_poisson_log_lpmf(array[] int, array[] int, vector,
+  vector, (T_k...) => matrix, tuple(T_k...), vector, data real, data int,
+  data int, data int, data int)
+However, we recieved the types:
+(data array[] int, data array[] int, data vector)
+Typechecking failed after checking the first 3 arguments. Please ensure you
+are passing enough arguments and that the 4th is a function.
 [exit 1]
   $ ../../../../../install/default/bin/stanc missing_args9.stan
 Semantic error in 'missing_args9.stan', line 40, column 24 to line 41, column 59:
    -------------------------------------------------
     38:  generated quantities {
     39:  
-    40:    vector[n_obs] theta = laplace_latent_tol_neg_binomial_2_log_rng(y, {1}, [1.0]',
+    40:    vector[n_obs] theta = laplace_latent_tol_neg_binomial_2_log_rng(y, {1}, [1.0]', [0]',
                                  ^
     41:                          K_function, (x, n_obs, alpha, rho));
     42:  
    -------------------------------------------------
 
-Ill-typed arguments supplied to function 'laplace_latent_tol_neg_binomial_2_log_rng'
-for the likelihood:
-The fourth argument must be vector but got <F1>.
-where F1 = (array[] vector, int, real, real) => matrix
-Expected the arguments to start with:
-(array[] int, array[] int, vector, vector)
+Recieved 0 control arguments at the end of the call to 'laplace_latent_tol_neg_binomial_2_log_rng'.
+Expected 6 arguments for the control parameters instead.
```

Comment:
Is this supposed to be different?

Reply:
Yes, you were looking only at the one commit diff. If you look at the overall diff of the PR you will see the message did not change.